pkg/provider/aci.go

```diff
@@ -1140,12 +1148,15 @@ func getProbe(probe *v1.Probe, ports []v1.ContainerPort) (*azaci.ContainerProbe,
 	// Probes have can have an Exec or HTTP Get Handler.
 	// Create those if they exist, then add to the
 	// ContainerProbe struct
-	var exec *azaci.ContainerExec
+	var commands []*string
```

Comment:
Do we have any test for this GetProbe method?

Reply:
Yes, a couple of UT. i.e. 
https://github.com/virtual-kubelet/azure-aci/pull/422/files/1158ba62e24a82ad54d0f11f8d78e6cb00bf7fa7#diff-4dfc3566ffba8db8d8aea43c319a37da8fe539574ed4a637bf11c737e446bbd8L663